lib/modules/manager/vendir/readme.md

```diff
@@ -0,0 +1,29 @@
+Renovate supports updating Helm Chart references in vendir.yml via the [vendir](https://carvel.dev/vendir/) tool. Renovate requires the presence of a [vendir lock file](https://carvel.dev/vendir/docs/v0.40.x/vendir-lock-spec/) which is generated by vendir and should be stored in source code.
+
+It supports both https and oci helm chart repositories.
+
+```yaml title="Example helmChart"
+# fetch Helm chart contents (optional; v0.11.0+)
+helmChart:
```

Comment:
use a more complete sample, helm chart should not be a root object inside vendir.